darwin/datatypes.py

```diff
@@ -1077,3 +1099,56 @@ def _maybe_add_bounding_box_data(data: Dict[str, UnknownType], bounding_box: Opt
 ExportParser = Callable[[Iterator[AnnotationFile], Path], None]
 
 ImportParser = Callable[[Path], Union[List[AnnotationFile], AnnotationFile, None]]
+
+
+class MaskTypes:
+    Palette = Dict[str, int]
+    Mode = Literal["index", "grey", "rgb"]
+    TypeOfRender = Literal["raster", "polygon"]
+    CategoryList = List[str]
+    ExceptionList = List[Exception]
+    UndecodedRLE = List[int]
+    DecodedRLE = List[List[int]]
+    ColoursDict = Dict[str, int]
+    RgbColors = List[int]
+    HsvColors = List[Tuple[float, float, float]]
+    RgbColorList = List[RgbColors]
+    RgbPalette = Dict[str, RgbColors]
+
+    RendererReturn = Tuple[ExceptionList, NDArray, CategoryList, ColoursDict]
+
+
+@dataclass
+class AnnotationMask:
```

Comment:
Not vital atm, but in v2 this could be a pydantic basemodel and would therefore have validation on creation and change without requiring explicit validate function

Reply:
It could, and atm I'm not gonna change it, but it's a good future change.